src/App/Resources/AppResources.resx

```diff
@@ -2512,4 +2512,7 @@ Do you want to switch to this account?</value>
   <data name="ThisRequestIsNoLongerValid" xml:space="preserve">
     <value>This request is no longer valid</value>
   </data>
+  <data name="EnableCamerPermissionsToUseTheScanner" xml:space="preserve">
+    <value>Enable camera permissions to use the scanner</value>
+  </data>
```

Comment:
Typo (Camer -> Camera) and I think permissions should be singular: `EnableCameraPermissionToUseTheScanner`